### DISCUSSION TITLE
Customise Branch line Colour and width

### DISCUSSION THREAD

**Discussion by viveknandhangelella** [react-querybuilder/react-querybuilder#524] [Q&A] [answered]
Hi Team, 

I am trying increase width and change default colour of the branch but I am not able override it. your help is much appreciated.
[sandBoxLink](https://codesandbox.io/p/sandbox/blazing-architecture-26mws1?file=%2Fsrc%2Fstyles.scss%3A14%2C34)

**Comment by jakeboone02** [comment, marked as answer]
Hi @viveknandhangelella! You need to remove the `!default` directive from your variable declarations and move them above the `@import` of the RQB SCSS.

Fixed example here: https://codesandbox.io/p/sandbox/dreamy-parm-4loez7?file=%2Fsrc%2Fstyles.scss%3A1%2C1-6%2C1

**Reply by viveknandhangelella** [reply]
thank you @jakeboone02  for quick reply i was able to add customise branch color

**Comment by viveknandhangelella** [comment]
thank you @jakeboone02 for quick reply i was able to add customise branch color
